habitat/config/default.py

```diff
@@ -910,9 +910,12 @@ def __init__(self, *args, **kwargs):
 # -----------------------------------------------------------------------------
 # Task
 # -----------------------------------------------------------------------------
-# Uless another RLEnv is implemented and registered, the default task is
-# should be RLTaskEnv. RLTaskEnv works for both Navigation and Rearrange.
-_C.ENV_TASK = "RLTaskEnv"
+# GymHabitatEnv works for all Habitat tasks, including Navigation and Rearrange.
+# To use a gym environment from the registry, use the GymRegistryEnv.
+# Any other environment needs to be created and registered.
+_C.ENV_TASK = "GymHabitatEnv"
+_C.ENV_TASK_DEPENDENCIES = []
```

Comment:
Can you add documentation for what this key does? That it imports all the modules you specify.

Reply:
Good call. Do you think this is a valid thing to do or should we require that the users import the libraries themselves and then register the environment ?

Reply:
Both are possible under this setup right? Users can either import their gym libraries manually or pass them into `_C.ENV_TASK_DEPENDENCIES`? I think `_C.ENV_TASK_DEPENDENCIES` makes life easier for habitat baselines "expert users" so it's a nice feature to have. Now you don't need to change the code at all and can easily import your new Gym environments.

Reply:
Ok sounds good. I will leave it here and rename to ENV_TASK_**GYM**_DEPENDENCIES to make it clear that it will only affect the gym environments. 

> Both are possible under this setup right? 

Yes, the user only needs to do 

```
@habitat.registry.register_env(name="MyEnvironment")
class MyEnvironment(gym.Wrapper):
    def __init__(self, config: Config, dataset: Optional[Dataset] = None):
        env = my_super_awesome_custom_environment_that_follows_the_gym_api()
        super().__init__(env)
```
And then make `ENV_TASK="MyEnvironment"`